### ISSUE TITLE
Single-point graphql constructor

### ISSUE THREAD

**Issue by Totktonada** [tarantool/graphql.0#59]
We should allow to omit explicit data accessor creation and provide graphql constructor receives all needed metainfo and data accessor name. It may look like that:

```
local gql_wrapper = graphql.new('space', {
    schemas = schemas,
    collections = collections,
    service_fields = service_fields,
    indexes = indexes,
}, {
   ...funcs if needed...
})
```

Reason: most of users just want to use one of on-board data accessors.

**Comment by SudoBobo** [comment]
What is the purpose of funcs here?
```
local gql_wrapper = graphql.new('space', {
    schemas = schemas,
    collections = collections,
    service_fields = service_fields,
    indexes = indexes,
}, {
   ...funcs if needed...
})
```
To overwrite 'space' or 'shard' functions like `unflatten_tuple` or `get_index`?

**Comment by Totktonada** [comment]
@SudoBobo yep.

**Comment by Totktonada** [comment]
@SudoBobo Can you please check whether this issue can be closed?

**Comment by SudoBobo** [comment]
Closed with  #95
